app/code/Magento/Sales/Test/Mftf/Test/AdminCreateOrderWithConfigurableProductPressKeyEnterTest.xml

```diff
@@ -9,6 +9,13 @@
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
     <test name="AdminCreateOrderWithConfigurableProductPressKeyEnterTest">
+        <annotations>
+            <title value="Create Order in Admin with configurable product"/>
+            <description value="Create order with configurable product."/>
```

Comment:
Could you please add "with pressing enter key in option select modal" to title and description to separate it with the normal flow ? Thanks

Reply:
thank @edenduong, I added new commit